homeassistant/components/deutsche_bahn/sensor.py

```diff
@@ -13,6 +13,8 @@
 
 CONF_DESTINATION = 'to'
 CONF_START = 'from'
+CONF_TIME_DELTA = 'offset'
```

Comment:
```suggestion
CONF_OFFSET = 'offset'
```

It's really confusing when the CONF_ name doesn't match the actual value.